<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>BlankCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QFrame" name="frameCurtainMenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>220</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <widget class="QFrame" name="frameBikeIndfo">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>35</y>
      <width>796</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lbBikeWheel">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>0</y>
       <width>311</width>
       <height>171</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="configuration.qrc">:/images/bike-rearwheel.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameslide">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>27</y>
      <width>264</width>
      <height>5</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="hmiMouseEvent" name="lbInfo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1</y>
      <width>265</width>
      <height>25</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="configuration.qrc">:/images/info.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="hmiMouseEvent" name="lbList">
    <property name="geometry">
     <rect>
      <x>267</x>
      <y>1</y>
      <width>266</width>
      <height>25</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="configuration.qrc">:/images/list.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="hmiMouseEvent" name="label">
    <property name="geometry">
     <rect>
      <x>534</x>
      <y>1</y>
      <width>265</width>
      <height>25</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="configuration.qrc">:/images/settings.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameMI">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>140</width>
     <height>60</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lbtrip">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="configuration.qrc">:/images/trip.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbtripValue">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>86.9</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbtripUnit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>mi.</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameKCAL">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>140</width>
     <height>60</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lbCalories">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="configuration.qrc">:/images/calories.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbCaloriesValue">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>2699</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbCaloriesUnit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>kcal</string>
    </property>
   </widget>
  </widget>
  <widget class="hmiMouseEvent" name="lbHighBeam">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="configuration.qrc">:/images/lights_off.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>450</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="hmiMouseEvent" name="lbCortainUpDown">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>150</width>
     <height>41</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="configuration.qrc">:/images/top_curtain_drag_back.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>450</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>hmiMouseEvent</class>
   <extends>QLabel</extends>
   <header location="global">hmimouseevent.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="configuration.qrc"/>
 </resources>
 <connections/>
</ui>
